website/screens/components/footer/code/examples/Icons.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
```

Comment:
Remove file, as it is not needed for basic usage example.

Reply:
Done